src/Common/Dialogs/DeleteDialog.tsx

```diff
@@ -21,13 +50,34 @@ export const DeleteDialog: React.FC<DeleteDialogProps> & { Description?: React.F
         props.closeDelete()
     }
 
+    const getLabel = () => (
+        <span>
+            Type
+            <span className="fw-6">&nbsp;&apos;{deleteConfirmationText}&apos;&nbsp;</span>
```

Comment:
Can add word break here

Reply:
added in https://github.com/devtron-labs/devtron-fe-common-lib/pull/171/commits/3718cfa1471e7e12d902eb5c46a7e867d0e8bd48